jobsdb/jobsdb.go

```diff
@@ -1647,11 +1670,12 @@ func (jd *HandleT) backupTable(backupDSRange dataSetRangeT, isJobStatusTable boo
 	// if backupOnlyAborted, process join of aborted rows of jobstatus with jobs table
 	// else upload entire jobstatus and jobs table from pre_drop
 	if jd.BackupSettings.AbortedOnly {
-		logger.Info("=======backing up aborted backuptable==========")
+		logger.Info("=======backing up aborted/failed entries==========")
```

Comment:
Change logging statement to [JobsDB] Backing up Aborted/failed entries

Reply:
done